File: stringComparator/stringComparator.java
package stringComparator;

public class stringComparator {
    public static boolean isStringEqual(String str1, String str2) {
        String string1 = str1.toLowerCase();
        String string2 = str2.toLowerCase();

        if (string1.equals(string2)) {
            return true;
        } else {
            return false;
        }
    }

    public static void main(String[] args) {
        String str1 = "kuda";
        String str2 = "kuDa";
        System.out.println(isStringEqual(str1, str2));
    }
}

File: stringComparator/stringComparator.js
let str1 = "kuda";
let str2 = "kuDa";

function stringComparator(str1, str2) {
  let str1Lower = str1.toLowerCase();
  let str2Lower = str2.toLowerCase();

  if (str1Lower === str2Lower) {
    return true;
  } else {
    return false;
  }
}

console.log(stringComparator(str1, str2));
